tools/debian/convert_debian.py

```diff
@@ -34,58 +34,118 @@
 # e.g. [buster] - xz-utils 5.2.4-1+deb10u1
 VERSION_PATTERN = re.compile(r'\[(.*?)\]\s*-\s*([^\s]+)\s*([^\s]+)')
 
+# TODO: Alternative is to use a xml parser here,
+#  though the data is not fully compliant with the xml standard
+#  It is possible to parse with an html parser however
+
+# e.g. <define-tag moreinfo>\n Some html here \n</define-tag>
+WML_DESCRIPTION_PATTERN = re.compile(
+    r'<define-tag moreinfo>((?:.|\n)*)</define-tag>', re.MULTILINE)
+
+# e.g. <define-tag report_date>2022-1-04</define-tag>
+WML_REPORT_DATE_PATTERN = re.compile(
+    r'<define-tag report_date>(.*)</define-tag>')
+
+# e.g. DSA-12345-2, -2 is the extension
+CAPTURE_DSA_WITH_NO_EXT = re.compile(r'dsa-\d+')
+
+
+class DebianSpecificInfo:
+    """Specific Debian information, exports to the
+    `ecosystem_specific` field in osv"""
 
-class VersionInfo:
-    """Debian version info."""
     release: str
+
+    def __init__(self, release: str):
+        self.release = release
+
+    def to_dict(self):
+        return self.__dict__
+
+    def __repr__(self):
+        return json.dumps(self, default=dumper)
+
+
+class AffectedInfo:
+    """Debian version info."""
+
+    ecosystem_specific: DebianSpecificInfo
     package: str
+    ranges: [str]
     fixed: str
+    versions: [str]
 
     def __init__(self, release: str, package: str, fixed: str):
-        self.release = release
+        self.ecosystem_specific = DebianSpecificInfo(release)
         self.package = package
         self.fixed = fixed
 
+    def to_dict(self):
+        return {
+            'ecosystem_specific':
+                self.ecosystem_specific,
+            'package': {
+                'ecosystem': 'Debian',
+                'name': self.package
+            },
+            'ranges': [{
+                'type': 'ECOSYSTEM',
+                'events': [{
+                    'fixed': self.fixed
+                }]
+            }],
+        }
+
     def __repr__(self):
-        return str({
-            'release': self.release,
-            'package': self.package,
-            'fixed': self.fixed,
-        })
+        return json.dumps(self, default=dumper)
 
 
 class AdvisoryInfo:
     """Debian advisory info."""
+
```

Comment:
Can you add a TODO for adding `modified` as well? We can probably take this from the git log.